dss/stepfunctions/visitation/reindex.py

```diff
@@ -58,49 +59,44 @@ def walker_finalize(self):
         self.marker = None
         self.token = None
 
-    def _walk(self, seconds_allowed=250) -> None:
-        start_time = time()
-
-        indexer_class = Indexer.for_replica[Replica[self.replica]]
-
-        self.indexer = indexer_class(dryrun=self.dryrun, notify=self.notify)
-
-        handle = Config.get_blobstore_handle(Replica[self.replica])
-        default_bucket = Replica[self.replica].bucket
-
-        if self.bucket != default_bucket:
-            logger.warning(f'Indexing bucket {self.bucket} instead of default {default_bucket}.')
-
-        blobs = handle.list_v2(
-            self.bucket,
-            prefix=f'bundles/{self.work_id}',
-            start_after_key=self.marker,  # type: ignore  # Cannot determine type of 'marker'
-            token=self.token  # type: ignore  # Cannot determine type of 'token'
-        )
-
-        for key in blobs:
-            seconds_remaining = int(seconds_allowed - (time() - start_time))
-
-            if 1 > seconds_remaining:
-                logger.info(f'{self.work_id} timed out before reindex')
-                return
-
-            with Timeout(seconds_remaining) as timeout:
-                """
-                Timing out while recording paging info could cause an inconsistent paging state, leading
-                to repeats of large amounts of work. This can be avoided by checking for timeouts only
-                during actual re-indexing.
-                """
-                self.process_item(key)
-
-            if timeout.did_timeout:
-                logger.warning(f'{self.work_id} timed out during reindex')
-                return
-
-            self.marker = blobs.start_after_key
-            self.token = blobs.token
-        else:
-            self._status = WalkerStatus.finished.name
+    def _walk(self) -> None:
+        backends = [ElasticsearchIndexBackend]
```

Comment:
nit: if this list and the list for the index call or there `CompositeIndexBackend` were centralized somewhere, it would decrease the chance that a future dev would add a new backend to the index app, but not to the reindex step function.

Reply:
Yup. Will fix.